.gitignore

```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+test.md
+node_modules
+licence.txt
```

Comment:
Please consider removing the LICENSE file from the .gitignore file as you need to have it on your repository(repo). Moreover, your README link to your MIT LICENSE file is not working and showing a 404 ERROR as there is no LICENSE file in your repo. Consider adding one.

![image](https://user-images.githubusercontent.com/94223367/218698498-205a4bd9-a06c-4827-ac2c-f5beed544a97.png)